src/pages/students/students.scheme.ts

```diff
@@ -0,0 +1,42 @@
+import * as yup from "yup";
+
+export const studentsScheme = yup
+  .object({
+    name: yup.string().required("Debe ingresar el nombre"),
+    lastname: yup.string().required("Debe ingresar el apellido"),
+    nroTelefono: yup
+      .string()
+      .min(8, "Debe tener al menos 8 caracteres")
+      .max(15, "No puede superar los 15 caracteres")
+      .matches(/^[+0-9\s-]+$/, "Solo se permiten números"),
```

Comment:
Aca te falta que el numero de telefono debe ser requerido

```suggestion
    nroTelefono: yup
      .string()
      .required("Debe ingresar un numero de telefono")
      .min(8, "Debe tener al menos 8 caracteres")
      .max(15, "No puede superar los 15 caracteres")
      .matches(/^[+0-9\s-]+$/, "Solo se permiten números"),
```